mozilla-release/mobile/android/app/src/main/res/layout/activity_stream_news_card.xml

```diff
@@ -1,5 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
-<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<!-- Copyright Cliqz © 2018 -->
```

Comment:
Is this an original file? Or is it derived from some Mozilla one?
We must be able to distinguish between our original content and Mozilla derived one.

Reply:
It doesn't seem to be derived from any files from Mozilla